### PR TITLE
Newspaper design

### PR DESCRIPTION
**Pull request by tiffz**
More on Tiff tries to design Saylua for a bit. I tried to spice up the Sayluan Gazette. 

screenshot 2017-02-17 01 24 11

### CODE REVIEW

**Comment by tiffz** [comment]
screenshot 2017-02-17 01 54 28

Latest screenshot. Good night world. :D

File: saylua/modules/home/templates/newspaper/news.html [hunk 1]
```diff
@@ -2,9 +2,22 @@
...
+    Explore in our map on Saylua
+  </h2>
+  <div class="news-post-attribution">by <a href="/user/">Tiff</a> on Feb 17, 2017 9:43PM SMT</div>
+  <article>
+    <p>
+      Check out the WIP explore feature here: <a href="/explore/">Explore</a>
+    </p>
+    <p>
+      Saylua is a pet about the beauty where the site conquers all. Virtualize your collections into
+      pets on this Smashing Hulk Hogan simulator. Loxi Vela ipsum.
```
Comment (NoiSek) [inline saylua/modules/home/templates/newspaper/news.html]: 10/10 would headbang again

**Review by NoiSek — APPROVED** [review]
This coverage guy is a real bully. Looks great, though. The white balance of all the new pages so far is particularly improved.

**Review by LikeMyBread — APPROVED** [review]
Simply gorgeous, looking wonderful!

**Pull request merged**